assets/scss/_alerts.scss

```diff
@@ -0,0 +1,5 @@
+// Style alert boxes.
```

Comment:
Could we not make an update/change to the site's theme independently. It would be nice to decouple them changes from content changes. Makes it much easier to review.